src/Lit.fs

```diff
@@ -26,3 +34,16 @@ let styles (styles: Feliz.CssStyle seq) =
     |> Seq.iter (fun (Feliz.CssStyle(key, value)) ->
         map?(key) <- value)
     Bindings.styleMap map
+    
+let classes (classes: (string * bool) seq) = 
+    Bindings.classMap (keyValueList CaseRules.LowerFirst classes)
+
+let repeat<'T> (items: 'T seq) (template: 'T -> int -> TemplateResult) = importMember "lit-html/directives/repeat"
```

Comment:
These don't really need an intermediary function hence why I pull them out of the bindings module, let me know if I should move those there

Reply:
I would prefer to put them in the Bindings so it's easier to move the bindings to a separate package if needed, I can do that :+1: